test/topology/test_topology.py

```diff
@@ -117,41 +160,46 @@ async def no_joining_nodes():
     await manager.api.enable_injection(
         bootstrapped_server.ip_addr, 'storage_service_decommission_prepare_handler_sleep', one_shot=True)
     await manager.decommission_node(decommission_target.server_id)
+    servers = await manager.running_servers()
+    await check_token_ring_and_group0_consistency(manager, servers)
     await table.add_column()
     await random_tables.verify_schema()
-    # TODO: check that group 0 no longer contains the decommissioned node (#12153)
 
 @pytest.mark.asyncio
 async def test_replace_different_ip(manager: ManagerClient, random_tables) -> None:
     servers = await manager.running_servers()
     await manager.server_stop(servers[0].server_id)
     replace_cfg = ReplaceConfig(replaced_id = servers[0].server_id, reuse_ip_addr = False, use_host_id = False)
     await manager.server_add(replace_cfg)
-    # TODO: check that group 0 no longer contains the replaced node (#12153)
+    servers = await manager.running_servers()
+    await wait_for_token_ring_and_group0_consistency(manager, servers, time.time() + 30)
```

Comment:
nit: all invocations of `wait_for_..._consistency` are preceded by the `await manager.running_servers()` call, so it's probably worth moving the latter call into the former

Reply:
Done